### PR TITLE
[loki-distributed] Add the extraVolumes/extraVolumeMounts/readinessProbe settings

### PR DESCRIPTION
**Pull request by diranged**
This PR standardizes the `extraVolumes` and `extraVolumeMounts` settings on ALL of the distributed loki pods, as well as allows for the customization of the `readinessProbe`. These two things are required in order to be able to pass in TLS certificates to the Loki processes (and Nginx gateway) to enable peer-to-peer encryption between the Loki nodes.

### CODE REVIEW

File: charts/loki-distributed/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: loki-distributed
 description: Helm chart for Grafana Loki in microservices mode
 type: application
 appVersion: 2.1.0
-version: 0.23.0
+version: 0.23.2
```
Comment (unguiculus) [inline charts/loki-distributed/Chart.yaml:6]: Please make this a minor bump, since you're adding new features.
```suggestion
version: 0.24.0
```

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: charts/loki-distributed/Chart.yaml (hunk 1 above)
Comment (diranged) [reply charts/loki-distributed/Chart.yaml:6]: Thanks - will do! Quick question, I find myself manually modifying the README.md. Is there a tool I should have installed to generate the automated version you guys have?

**Comment by diranged** [comment]
@unguiculus I have cleaned up the PR and removed a bit of accidental cruft from my testing. I think it's good to go. We're actually using this now and it's working great.

File: charts/loki-distributed/Chart.yaml (hunk 1 above)
Comment (unguiculus) [reply charts/loki-distributed/Chart.yaml]: docker run --rm --volume "$(pwd):/helm-docs" -u "$(id -u)" jnorwood/helm-docs:v1.4.0

**Review by unguiculus — APPROVED** [review]

**Review by unguiculus — CHANGES_REQUESTED** [review]
Docs will have to be fixed. Run this and commit:

```
docker run --rm --volume "$(pwd):/helm-docs" -u "$(id -u)" jnorwood/helm-docs:v1.4.0
```

**Review by unguiculus — APPROVED** [review]

**Comment by diranged** [comment]
Thanks for merging this!!

**Pull request merged**